php-practice/section7.php

```diff
@@ -76,3 +97,15 @@
  *
  * (It returns TRUE if $needle is a substring of $haystack; FALSE otherwise.)
  */
+
+function stringContains($haystack, $needle) {
+	$pos = strpos($haystack, $needle);
+
+		if ($pos == true) {
+			echo "TRUE";
+		} else {
+			echo "FALSE";
+		}
+}
```

Comment:
Good. Again, note that the instructions want you to `return true;` or `return false;`, not print it out to the screen. Otherwise, this is correct!

I noticed that the comparison on line 104 is `$pos == true`. Do you know why this works, despite the fact that `strpos` returns an integer? (Not a boolean)